src/style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir-Book.ttf) format("ttf");
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.bigImg,
+.smallImg {
+  position: absolute;
+  color: #687c94;
+}
+
+.bigImg {
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.smallImg {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.bigSearch,
+.smallSearch {
+  position: relative;
+  margin-top: 20px;
+}
+
+.bigSearch input {
+  line-height: 67px;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: unset;
+  font-weight: 300;
+  font-size: 16px;
+  padding: 1px 2px 0 62px;
+}
+
+.smallSearch input {
+  line-height: 39px;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: unset;
+  font-weight: 400;
+  font-size: 14px;
+  padding: 1px 2px 0 33px;
```

Comment:
check out point 2 in the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)